src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's good that you have 'Arial, sans-serif' as fallbacks, but make sure to include them for all font declarations if you have others in your CSS.